### PR TITLE
chore: Add docstrings

### PR DESCRIPTION
**Pull request by felixwang9817**
**What this PR does / why we need it**: These docstrings are necessary for RTD.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py [hunk 1]
```diff
@@ -97,6 +97,24 @@ def __init__(
         tags: Optional[Dict[str, str]] = None,
         owner: Optional[str] = "",
     ):
+        """
+        Creates a TrinoSource object.
+
+        Args:
+            name (optional): Name for the source. Defaults to the table if not specified.
```
Comment (achals) [inline sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py:104]: What happens if a query is specified instead?

File: sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py [hunk 2]
```diff
@@ -97,6 +97,24 @@ def __init__(
...
+            timestamp_field (optional): Event timestamp field used for point in time
+                joins of feature values.
+            table (optional): Trino table where the features are stored.
+            created_timestamp_column (optional): Timestamp column indicating when the
+                row was created, used for deduplicating rows.
+            field_mapping (optional): A dictionary mapping of column names in this data
+                source to column names in a feature table or view.
+            query (optional): The query to be executed to obtain the features.
```
Comment (achals) [inline sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py:112]: Maybe we should be explicit that one of `table` or `query` should be specified.

**Review by achals — APPROVED** [review]

**Review by kevjumba — APPROVED** [review]
/lgtm

File: sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py (hunk 1 above)
Comment (felixwang9817) [reply sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py:104]: in that case, the `name` must be specified

Comment (achals) [reply sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py:104]: Update the doc string to reflect that?

**Review by achals — APPROVED** [review]
/lgtm

**Pull request merged**